src/hotspot/os_cpu/windows_aarch64/thread_windows_aarch64.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Microsoft Corporation. All rights reserved.
+ * Copyright (c) 2020, 2022 Microsoft Corporation. All rights reserved.
```

Comment:
If I remember it correctly, we'd better not modify the copyright line other than Oracle's.
Maybe, you can add a new line here for Alibaba?

Reply:
Thanks for the reminder. 
I added a new line for Alibaba for this file.

Reply:
> Thanks for the reminder. I added a new line for Alibaba for this file.

Thanks for the update.

Well, I would suggest keeping Microsoft Corporation the first line, then Alibaba since the file was created by Microsoft.
What do you think?

Reply:
Make sense, updated.